sequencer/src/state.rs

```diff
@@ -202,15 +221,24 @@ pub fn validate_proposal(
         )
     );
 
+    // validate block size and fee
+    let block_size = VidSchemeType::get_payload_byte_len(vid_common) as u64;
     anyhow::ensure!(
-        (VidSchemeType::get_payload_byte_len(vid_common) as u64)
-            < expected_chain_config.max_block_size(),
+        block_size < expected_chain_config.max_block_size,
```

Comment:
This was added some time ago but wouldn't it be more intuitive to make the range inclusive so that a block of `max_block_size` is still allowed? If yes, let's add an issue.

Reply:
https://github.com/EspressoSystems/espresso-sequencer/issues/1464